template/_package.json

```diff
@@ -21,6 +21,11 @@
     "dev": "nodemon --watch app --watch bootstrap --watch config --watch .env -x node server.js",
     "build": "cross-env NODE_ENV=production node ./ace nuxtbuild",
     "start": "cross-env NODE_ENV=production node server.js"
+    <%_ } else if (server === 'express') { _%>
```

Comment:
I think this part can be same as none part

Reply:
agree

Reply:
What do you mean? The only way to correctly start nuxt in which it reports build progress in browser is AFAIK `--exec nuxt dev`. You can't start nuxt with server middleware by invoking `node server/index.js`.

Reply:
I've reread your comments. There's problems with using existing configs for express.

With server === 'none' it just runs nuxt dev, without watching server folder. In that case it'll not update server files if you change them, because currently nuxt doesn't support recompilation of serverMiddleware.
If you use last `else` with `"cross-env NODE_ENV=development nodemon server/index.js --watch server",` it'll not work with server middleware, since we need to start nuxt, node file.

So if you want to use default option for express, you need to start nuxt with nodemon, I'm not sure it's needed if you don't have server middleware.

Reply:
`serverMiddleware` has already been watch since nuxt 2.7 which is from https://github.com/nuxt/nuxt.js/pull/5681

Reply:
I checked it while making this pull request and changing content of server/index.js makes nuxt recompile something, but doesn't actually change result of request to express middleware.

Reply:
@Atinux

Reply:
Okay, so I've made couple more tests.
![image](https://user-images.githubusercontent.com/5431520/62006449-4cc1a980-b149-11e9-87cf-9a007c82968f.png)
If you export just express instance hot reloading works.
If you export object with path and handler hot reloading doesn't work.
I'll update pull. Maybe we should create issue in nuxt repository.

Reply:
![image](https://user-images.githubusercontent.com/5431520/62006729-433a4080-b14d-11e9-959d-587da72dcd35.png)
Turns out outside from exporting app, you also need to explicitly include that file.